server/controllers/cv-controller.js

```diff
@@ -0,0 +1,17 @@
+
+const clinet = require('../lib/github-client.js');
+
+const hundleRequest = (name, res) => {
+  const userPromise = clinet.getUserProfile(`${name}`);
+  const userPullrequest = clinet.getUserPullRequests(`${name}`);
+  Promise.all([userPromise, userPullrequest])
+  .then((userData) => {
+    const numberOfPullReq = userData[1].length;
+    res.render('myCv', { actual: userData[0], numberPullRequest: numberOfPullReq });
+  }).catch(reason => console.log('Promise rejected : ', reason));
+};
+
+module.exports.route1 = (req, res) => {
```

Comment:
this is a really bad name for a function. Rather than exporting this as an object, you could just export the function
```js
module.exports = (req, res) => {
  ...
};
```
and then in your `app.js` you would do
```js
const cvController = require('./controllers/cv-controller');
...
app.get('/', cvController);
```